guake/keybindings.py

```diff
@@ -82,6 +82,7 @@ def x(*args):
             ("decrease-transparency", self.guake.accel_decrease_transparency),
             ("toggle-transparency", self.guake.accel_toggle_transparency),
             ("search-on-web", self.guake.search_on_web),
+            ("open-link-under-terminal-cursor", self.guake.open_link_under_terminal_cursor),
```

Comment:
A bit weird that this is in the middle like this but doesn't really matter I guess

Reply:
yeah agree doesn't matter much, just thought it fit better next to the other "Extras" category binding, than next to the more "terminal" related ones.